onnxruntime/test/python/transformers/conformer_model_generator.py

```diff
@@ -0,0 +1,481 @@
+        "MatMul_q_weight",
+        TensorProto.FLOAT,
+        [hidden_size, hidden_size],
+        q_weight_data,
+    )
+    matmul_k_weight = helper.make_tensor(
+        "MatMul_k_weight",
+        TensorProto.FLOAT,
+        [hidden_size, hidden_size],
+        k_weight_data,
+    )
+    matmul_v_weight = helper.make_tensor(
+        "MatMul_v_weight",
+        TensorProto.FLOAT,
+        [hidden_size, hidden_size],
+        v_weight_data,
+    )
+    qkv_bias = helper.make_tensor(
+        "Attention_0_qkv_bias",
+        TensorProto.FLOAT,
+        [3 * hidden_size],
+        q_bias_data + k_bias_data + v_bias_data,
+    )
+    initializers = [
+        float_tensor("layernorm_weight", [hidden_size]),
+        float_tensor("layernorm_bias", [hidden_size]),
+        float_tensor("matmul_after_attn_initializer", [hidden_size, hidden_size]),
+        float_tensor("add_after_attn_initializer", [hidden_size]),
+        helper.make_tensor("concat_reshape", TensorProto.INT64, [4], [1, 8, 8, 80]),
+    ]
+
+    # Add Q/K/V weight tensors as initializers
+    if fused:
+        initializers.extend([matmul_q_weight, matmul_k_weight, matmul_v_weight])
+        initializers.append(qkv_bias)
+    else:
+        initializers.extend([q_weight, k_weight, v_weight])
+
+        initializers.extend([q_bias, k_bias, v_bias])
+
+        initializers.extend(
+            [
+                numpy_helper.from_array(np.array(num_heads, dtype="int64"), name="num_heads_int"),
+                numpy_helper.from_array(np.array([num_heads], dtype="int64"), name="num_heads"),
+                numpy_helper.from_array(np.array([head_size], dtype="int64"), name="head_size"),
+                numpy_helper.from_array(np.array([hidden_size], dtype="int64"), name="hidden_size"),
+                numpy_helper.from_array(np.array(1 / np.sqrt(head_size), dtype="float32"), name="q_scale"),
+                numpy_helper.from_array(np.array(0, dtype="int64"), name="idx_0"),
+                numpy_helper.from_array(np.array(1, dtype="int64"), name="idx_1"),
+                numpy_helper.from_array(np.array([-1], dtype="int64"), name="neg_one"),
+                numpy_helper.from_array(np.array([0], dtype="int64"), name="unsqueeze_axes_input"),
+            ]
+        )
+
+    # Construct graph
+    graph = helper.make_graph(nodes, "ct_self_mha_graph", inputs, outputs, initializers, doc_string="conformer")
+    opsetid = helper.make_opsetid("ai.onnx", min(onnx.defs.onnx_opset_version(), 16))
+    return helper.make_model(graph, opset_imports=(opsetid,))
+
+
+if __name__ == "__main__":
+    np.random.seed(2)
+    num_heads = 8
+    hidden_size = 512
+
+    model = create_conformer_attention(num_heads=num_heads, hidden_size=hidden_size)
+    onnx.save(model, "conformer_self_mha.onnx")
+
+    model = create_conformer_attention(num_heads=num_heads, hidden_size=hidden_size, fused=True)
+    onnx.save(model, "./test_data/models/conformer/conformer_self_mha_fused.onnx")
```

Comment:
You can update `cmake/onnxruntime_python.cmake` so that the CI pipelines can see this ONNX model.

There are several commands to add in the `cmake` file. Ex:
https://github.com/microsoft/onnxruntime/blob/02333293dec94922585a4aed39bd331128b643a6/cmake/onnxruntime_python.cmake#L436-L438
https://github.com/microsoft/onnxruntime/blob/02333293dec94922585a4aed39bd331128b643a6/cmake/onnxruntime_python.cmake#L550
https://github.com/microsoft/onnxruntime/blob/02333293dec94922585a4aed39bd331128b643a6/cmake/onnxruntime_python.cmake#L701-L703

Reply:
Done updated